src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotResolvedJavaMethod.java

```diff
@@ -58,6 +58,13 @@ public interface HotSpotResolvedJavaMethod extends ResolvedJavaMethod {
      */
     boolean hasReservedStackAccess();
 
+    /**
+     * Returns true if this method has a {@code Scoped} annotation.
```

Comment:
Please spell out where exactly the annotation is defined to avoid confusion with the unrelated concept of `java.lang.ScopedValue`:
```
     * Returns true if this method has a {@code jdk.internal.misc.ScopedMemoryAccess.Scoped} annotation.
```